taxcrunch/multi_cruncher.py

```diff
@@ -84,14 +84,31 @@ def read_input(self):
             ivar = self.path
         else:
             ivar = pd.read_csv(self.path, sep=',', engine="python", header=None)
-
         # translate INPUT variables into OUTPUT variables
         c = cr.Cruncher()
         self.invar = c.translate(ivar)
         self.rows = len(self.invar.index)
         return self.invar, self.rows
 
     def create_table(self, reform_file=None):
+        pol = self.get_pol(reform_file)
+        year = self.invar['FLPDYR'][0]
+        year = year.item()
+        recs = tc.Records(data=self.invar, start_year=year)
+        calc = tc.Calculator(policy=pol, records=recs)
+        calc.advance_to_year(year)
+        calc.calc_all()
+        calcs = calc.dataframe(self.tc_vars)
+        mtr = calc.mtr(wrt_full_compensation=False)
+        mtr_df = pd.DataFrame(data=mtr).transpose()
+        df_res = pd.concat([calcs, mtr_df], axis=1)
+        df_res.columns = self.labels
+        df_res.index = range(self.rows)
+        return df_res
+
+    def get_pol(self, reform_file):
+       
+        # if a reform file is not specified, the default policy is current law
         """
```

Comment:
Can you move this docstring to the `create_table()` method and create a new docstring for the `get_pol()` method?